Added Request Object for Error Logging

prashant-shahi opened the pull request.

prashant-shahi committed:
Error Logging includes Request Object

prashant-shahi committed:
Updated README.md

abraarsyed commented on README.md:
```diff
@@ -6,18 +6,27 @@ This is simple demonstration project on how [Nats.io](https://nats.io/) can be i
 2. [NPM - Node Package Manager](https://www.npmjs.com/)
 3. [Nats.io](https://nats.io/)
 
+### Build, Setup and Run using script file
```
Run script is not required because not all the users will have docker installed

abraarsyed commented on README.md:
````diff
@@ -6,18 +6,27 @@ This is simple demonstration project on how [Nats.io](https://nats.io/) can be i
 ```
 
-# Setting up the project
+### Setting up the project
````
This is not required. Move back to the old heading style. This is because the docker setup is just quick trick but not the only way for nats to be installed

abraarsyed commented on app.js:
```diff
@@ -9,13 +9,34 @@ app.use(bodyParser.urlencoded({ extended: true }));
+  
   if (err) {
-    errorhandler.handleError(err);
+
```
Empty lines not required inside the function

abraarsyed reviewed (changes requested):
Can you make the required changes and resubmit?

prashant-shahi committed:
Resolved as required

- Removed run script
- Reverted back to old README.md
- Eliminated empty lines inside the function

prashant-shahi commented:
Resolved as required.
